.github/workflows/test.yml

```diff
@@ -23,7 +23,8 @@ jobs:
           - stable-2.13
           - stable-2.14
           - stable-2.15
-          - devel
+          - stable-2.16
+          - stable-2.17
```

Comment:
We should look for no longer maintained ansible versions to drop.

Reply:
What about the python versions?
https://docs.ansible.com/ansible/latest/reference_appendices/release_and_maintenance.html#ansible-core-support-matrix
There are tests run for python 3.6 and 3.7 with ansible 2.11

Can we just go this way:
```
      matrix:
        python:
          - "3.10"
          - "3.11"
          - "3.12"
        ansible:
          - stable-2.16
          - stable-2.17
        include:
          - python: "3.12"
            ansible: "devel"
            image_tag: "nightly"
            lower_bounds: true

```

Reply:
Let's have a look at:
https://docs.ansible.com/ansible/latest/reference_appendices/release_and_maintenance.html#ansible-core-support-matrix

We should have a runner for ansible-core 2.15-2.17 + devel and spread python 3.9 - 3.12 over it.
The image_tag enables to run livetests in that runner too.
Also I would try to make the matrix as sparse as possible. Maybe we can reduce it to 4 combinations.

Reply:
I took the liberty incorporating your work into my dependency version bump:
https://github.com/pulp/squeezer/pull/167

I hope you don't mind.

Reply:
Not at all.
I think we have all scenarios covered now. I rebased and squashed. Please review so I can continue with fixing the timeout issue #164.